src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
It looks like there's a syntax error here due to a missing closing quotation mark at the end of the string. This will prevent the code from running. Make sure to close the string with a matching quotation mark.